src/rpm/package.rs

```diff
@@ -590,6 +590,56 @@ impl RPMPackageMetadata {
         )
     }
 
+    /// An RPM package is comprised of several segments - the Lead, Signature Header, Header, and Payload.
+    /// This function returns the computed (byte) boundaries between those segments in the package file
+    /// as if it were written out on-disk.
+    ///
+    /// ```
+    /// # use rpm::RPMPackage;
+    /// # let package = RPMPackage::open("test_assets/389-ds-base-devel-1.3.8.4-15.el7.x86_64.rpm").unwrap();
+    /// let offsets = package.metadata.get_package_segment_offsets();
+    /// let lead = offsets.lead..offsets.signature_header;
```

Comment:
What I don't necessarily love about the struct is that if you want to construct a range then the order is entirely implicit, unlike the struct.

Of course we could return ranges, but ranges in Rust kind of suck, because they're not `Copy`

Reply:
@drahnr

Reply:
I personally would prefer the requirement to understand the boundary names over unnamed tuple fields.

Reply:
Does the PR look ok then?